malasakit-django/pcari/templates/personal-information.html

```diff
@@ -82,10 +84,10 @@
     </tr>
     <tr>
       <td>{% trans 'Barangay' %}</td>
-      <td><input id="barangay"></td>
+      <td><input id="barangay" type="text-field"></td>
```

Comment:
`text-field` is not an [input type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_<input>_types). What you're looking for is `type="text"`.